packages/SwingSet/test/test-terminate.js

```diff
@@ -0,0 +1,18 @@
+import '@agoric/install-ses';
```

Comment:
Could we move this test into `test/vat-admin/` somewhere? That's where we have all the other dynamic vat tests.

We could name the vat files with things like `terminate-bootstrap.js` and `terminate-vat-dude.js` if you wanted to avoid creating an extra basedir within there. I'm still iterating on what the cleanest file/dir layout would be, especially now that JSON config files are an option, so feel free to use a different approach.

Reply:
Swingset config files are a good option for allowing us to comingle sources for a bunch of test vats in a common directory (and this test already has such a config file) but I think that's worth doing only when the various different tests share some of the their vat code.  In this case they don't.  It's possible that as we're able to test more cases w.r.t vat termination that we'll end up with some points of commonality with the existing vat admin tests.  I'd wait for that.